Examples/Detectors/TelescopeDetector/include/ActsExamples/TelescopeDetector/TelescopeDetectorOptions.hpp

```diff
@@ -0,0 +1,53 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2020 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Units.hpp"
+#include "ActsExamples/Utilities/Options.hpp"
+
+#include <utility>
+
+#include <boost/program_options.hpp>
+
+namespace ActsExamples {
+
+namespace Options {
+
+namespace po = boost::program_options;
+
+/// The telescope geometry options
+///
+/// @tparam options_t Type of the options object (bound to boost API)
+///
+/// @param opt The provided object, where root specific options are attached
+template <typename options_t>
+void addTelescopeGeometryOptions(options_t& opt) {
+  opt.add_options()(
+      "geo-tele-longshifts",
+      po::value<read_range>()->multitoken()->default_value(
+          {0, 30, 60, 120, 150, 180}),
+      "Telescope detector Input: the layers shifts in the longidutinal "
+      "direction [mm], i.e. shifts along x, y or z axis")(
+      "geo-tele-transhifts",
```

Comment:
I am not sure the names are clear enough, maybe `geo-tele-positions` and `geo-tele-offsets`?

Reply:
Changed.